libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts

```diff
@@ -4,6 +4,8 @@ import { Widget } from "./widget";
 
 import { WidgetDropZone } from "./enum/DropZone";
 
+const MAXIMUM_TIMEOUT = Cypress.env("timeForInsightLoading");
```

Comment:
the value get from Cypress.env should move to support/constants.ts, and use from a get function. Please help to do the same with the const MAXIMUM_TIMEOUT on widget.ts file too